app/package.json

```diff
@@ -15,11 +15,13 @@
     "storybook": "start-storybook -p 6006",
     "storybook-build": "build-storybook",
     "test": "jest --ci --coverage",
+    "test-e2e": "playwright test",
     "ts-check": "tsc --noEmit"
   },
   "dependencies": {
     "@uswds/uswds": "^3.1.0",
     "i18next": "^21.9.2",
+    "lodash": "^4.17.21",
```

Comment:
just a note (you may already know this 🙏 ) but lodash can be gnarly for increasing bundle size - you can reference something like this to see about native js alternatives: https://youmightnotneed.com/lodash/ I am forgetting off-hand if there is a good alternative for deep clone which I think you said is your use case

*anyways* if it's useful could dig up more resources around this - IIRC you can use like webpack bundle analyzer or something like that to help reduce share of your bundle.

Reply:
I did not know, but should have guessed! Thank you for letting me know about this! At the time when I introduced lodash, I looked into native js alternatives and was surprised that the most common one was to use `JSON.parse(JSON.stringify(thingToDeepClone))`, which seemed ... 🥴 so I chose lodash instead, but maybe that's the better option?